documentation/2.0/content/rcuinfo.md

```diff
@@ -161,3 +182,205 @@ domainInfo:
+        opss-audit-viewDS:
+            Target: cluster-1,admin-server
+            JdbcResource:
+                JDBCConnectionPoolParams:
+                    TestTableName: SQL ISVALID
+                JDBCDataSourceParams:
+                    GlobalTransactionsProtocol: None
+                    JNDIName: jdbc/AuditViewDataSource
+                JDBCDriverParams:
+                    URL: --FIX ME--
+                    PasswordEncrypted: --FIX ME--
+                    DriverName: oracle.jdbc.OracleDriver
+                    Properties:
+                        user:
+                            Value: FMW1_IAU_VIEWER
+        opss-audit-DBDS:
+            Target: cluster-1,admin-server
+            JdbcResource:
+                JDBCConnectionPoolParams:
+                    TestTableName: SQL ISVALID
+                JDBCDataSourceParams:
+                    GlobalTransactionsProtocol: None
+                    JNDIName: jdbc/AuditAppendDataSource
+                JDBCDriverParams:
+                    URL: --FIX ME--
+                    PasswordEncrypted: --FIX ME--
+                    DriverName: oracle.jdbc.OracleDriver
+                    Properties:
+                        user:
+                            Value: FMW1_IAU_APPEND
+        mds-owsm:
+            Target: cluster-1,admin-server
+            JdbcResource:
+                JDBCConnectionPoolParams:
+                    InitialCapacity: 0
+                    TestConnectionsOnReserve: true
+                    ConnectionCreationRetryFrequencySeconds: 10
+                    TestTableName: SQL ISVALID
+                    TestFrequencySeconds: 300
+                    SecondsToTrustAnIdlePoolConnection: 0
+                JDBCDataSourceParams:
+                    GlobalTransactionsProtocol: None
+                    JNDIName: jdbc/mds/owsm
+                JDBCDriverParams:
+                    URL: --FIX ME--
+                    PasswordEncrypted: --FIX ME--
+                    DriverName: oracle.jdbc.OracleDriver
+                    Properties:
+                        user:
+                            Value: FMW1_MDS
+                        oracle.net.CONNECT_TIMEOUT:
+                            Value: '120000'
+                        SendStreamAsBlob:
+                            Value: 'true'
+```
+
+
+#### Oracle Active Grid Link Data Source (AGL)
+
+For setting data source to access Oracle Active Grid Link database, besides updating the `URL`, `PasswordEncrypted`, and the `user` property value; 
+you can specify additional `JDBCOracleParams` under `JdbcResource` of each data source.  For example,
+
+```yaml
+      mds-owsm:
+            Target: cluster-1,admin-server
+            JdbcResource:
+              JDBCOracleParams:
+                OnsNodeList:  'node1, node2'
+                ....
+```
+
+You can run WDT tool `modelHelp.sh -oracle_home <oracle home> resources:/JDBCSystemResource/JdbcResource/JDBCOracleParmas` for the complete list 
```

Comment:
re-write: For the complete list of fields, run the WDT command, `modelHelp.sh -oracle_home <oracle home> resources:/JDBCSystemResource/JdbcResource/JDBCOracleParmas`.